### ISSUE TITLE
Convert the Manifesto & Declaration PDFs into Markdown blog posts

### ISSUE THREAD

**Issue by hitchcott** [ethereumclassic/ethereumclassic.github.io#304]
Right now we're serving the translations for many language via PDF files.

https://github.com/ethereumclassic/ethereumclassic.github.io/tree/source/static

This is clunky and bad for SEO.

We should ideally replace them with actual markdown blog posts and remove the PDFs as each language version is released.

**Comment by TheEnthusiasticAs** [comment]
agree. I will remove the links to pdfs to other languages in those blogs in my translations.

**Comment by gitr0n1n** [comment]
https://etherplan.com/2019/06/17/the-crypto-decentralist-manifesto-in-more-languages/7885/

@TheEnthusiasticAs this might be helpful.

**Comment by gitr0n1n** [comment]
Oh nevermind I see the issue. Did you replace the .pdf's @TheEnthusiasticAs The Declaration repo seems to be broken as well.

**Comment by gitr0n1n** [comment]
Added to wiki:
https://github.com/ethereumclassic/wiki/wiki/The-Ethereum-Classic-Declaration-Of-Independence

https://github.com/ethereumclassic/wiki/wiki/A-Crypto-Decentralist-Manifesto

**Comment by gitr0n1n** [comment]
It appears this issue was completed and can be closed. Here are the links for review.

Manifesto: https://ethereumclassic.org/blog/2016-07-11-crypto-decentralist-manifesto

Declaration: https://ethereumclassic.org/blog/2016-08-13-declaration-of-independence

**Comment by hitchcott** [comment]
Whilst the english version is indeed a blog post, this issue is for the translated versions.

Those are still in PDFs linked at the top, and once those other languages are enabled, they should be converted.

**Comment by gitr0n1n** [comment]
ah i see. my bad, i thought we have an auto translator

**Comment by TheEnthusiasticAs** [comment]
The auto-translator delivers sometimes illogical translations
